apps/web/src/views/Notifications/containers/OnBoardingView.tsx

```diff
@@ -51,35 +53,53 @@ function OnboardingButton({ onClick, loading, isOnBoarded, account, isReady }: I
   )
 }
 
-const OnBoardingView = ({ identityKey, handleRegistration, isReady }: IOnBoardingProps) => {
+const OnBoardingView = ({
+  identityKey,
+  handleRegistration,
+  isReady,
+  isSubscribing,
+  setIsSubscribing,
+}: IOnBoardingProps) => {
+  const [isOboarding, setIsOnBoarding] = useState<boolean>(false)
   const toast = useToast()
   const { t } = useTranslation()
   const { address: account } = useAccount()
-  const { subscribe, isSubscribing } = useManageSubscription(`eip155:1:${account}`)
-  const { sendPushNotification, requestNotificationPermission, subscribeToPushNotifications } =
-    useSendPushNotification()
+  const { subscribe } = useManageSubscription(`eip155:1:${account}`)
+  const { sendPushNotification, subscribeToPushNotifications } = useSendPushNotification()
 
   const handleSubscribe = useCallback(async () => {
+    setIsSubscribing(true)
     try {
-      await subscribe()
       await subscribeToPushNotifications()
+      await subscribe()
       setTimeout(async () => {
         await sendPushNotification(BuilderNames.OnBoardNotification, [], `eip155:1:${account}`)
```

Comment:
why here use eip155 but fixed chainId to `1`?

Reply:
its a convention with wallet connect the way they store addresses in their DB. basically they store subscribers as `eip155:${chainId}:0x...` so what this means is that if we set address in this format then user subscriptions would be bound by the chainid, meaning that users would have to go through the onboarding process on each chain. 

furthermore this means that user notifications would be dependant on chain. if you wanted to see your binance notifications you would need to switch to binance, if you wanted to see your ethereum notifications you would need to switch to main net ethereum. also this means that you wouldnt say you were currently on ethereum and you got a binance notification. you would get notified but wouldn't be able to see the notification or have any way to know that it came from binance unless you just happened to switch chains.

this format from the WC team will be phased out and from talking to their lead engineer who i communicate with daily, this work around is acceptable